packages/remix/package.json

```diff
@@ -34,7 +34,8 @@
   },
   "devDependencies": {
     "@remix-run/node": "^1.4.3",
-    "@remix-run/react": "^1.4.3"
+    "@remix-run/react": "^1.4.3",
+    "portfinder": "^1.0.28"
```

Comment:
Why is this added here if we're no longer using it?

Reply:
We're still using it in remix integration tests, we just forgot to add it as a dependency there.